Update product-technical-writing-team-handbook.md

justinegeffen opened the pull request:
Removed customer education processes and details; updated AORs.

justinegeffen committed:
Update product-technical-writing-team-handbook.md

Removed customer education processes and details; updated AORs.

justinegeffen commented on operations/research-and-development/product/product-technical-writing-team-handbook.md:
```diff
@@ -6,7 +6,7 @@ description: 1% Draft
 ## Technical Writing Handbook
 
-The Mattermost Technical Writers focuses primarily on Technical Documentation (including the User's Guide, Administrator's Guide, and Developer documentation) and on Customer Education content.
+The Mattermost Technical Writers focus primarily on Technical Documentation (including the User's Guide, Administrator's Guide, and Developer documentation) and on Customer Education content.
```
```suggestion
The Mattermost Technical Writers focus primarily on writing, editing, and maintaining technical documentation (including the User's Guide, Administrator's Guide, and Developer documentation).
```

justinegeffen committed:
Update operations/research-and-development/product/product-technical-writing-team-handbook.md

justinegeffen commented on operations/research-and-development/product/product-technical-writing-team-handbook.md:
```diff
@@ -24,22 +24,10 @@ The Mattermost Technical Writers focuses primarily on Technical Documentation (i
-  - Attend customer calls undergoing enterprise onboarding
-- Tone and writing style guide for Customer Education content
-
 The following areas remain a collaborative effort between Product Managers, Engineers, and Technical Writers, until additional headcount is reached. Please reach out to the [Documentation Working Group](https://community.mattermost.com/private-core/channels/dwg-documentation-working-group) for any questions.
```
```suggestion
The following areas remain a collaborative effort between Product Managers, Engineers, and Technical Writers until additional headcount is reached. Please reach out to the [Documentation Working Group](https://community.mattermost.com/private-core/channels/dwg-documentation-working-group) for any questions.
```

justinegeffen commented on operations/research-and-development/product/product-technical-writing-team-handbook.md:
```diff
@@ -71,7 +59,7 @@ Once complete, submit your Pull Request \(PR\). Ensure that you assign appropria
 Feature documentation is a joint effort between Product Managers and Technical Writers. In the same way that we want to empower everyone to contribute to our documentation, Product Managers are encouraged to write MVP documentation for their product/feature.
 
-**Note**: Due to the cadence of the release cycle, feature documentation needs to be complete and submitted as per the process below at least ten days prior to release to ensure it's included in the documentation update.
+**Note:** Due to the cadence of the release cycle, feature documentation needs to be complete and submitted as per the process below at least ten days prior to release to ensure it's included in the documentation update.
```
```suggestion
**Note:** Due to the cadence of the release cycle, feature documentation needs to be complete and submitted as per the process below at least ten days prior to release to ensure it's included in the release documentation update.
```

justinegeffen commented on operations/research-and-development/product/product-technical-writing-team-handbook.md:
```diff
@@ -209,7 +197,7 @@ If the Jira ticket is for general documentation, depending on the current worklo
 ### Editing Requests
 
-The Technical Product Writer/s form part of the PR review/approval process. When submitting a PR for documentation, please add the **Editor Review** label. Once the editor has signed off on the PR they will remove the label. When all the requested reviews are complete, the **Reviews Complete** label will be applied and the changes merged.
+When submitting a PR for documentation, please add the **Editor Review** label. Once the Technical Writer/Editor has signed off on the PR they will remove the label. When all the requested reviews are complete, the **Reviews Complete** label will be applied and the changes merged.
```
```suggestion
When submitting a PR for documentation, whether it's a minor update, a new piece of content, or a content proposal, please add the **Editor Review** label. Once the Technical Writer/Editor has reviewed the PR they'll remove the label. When all the requested reviews are complete, the **Reviews Complete** label is applied and the changes are merged.
```

justinegeffen committed:
Update operations/research-and-development/product/product-technical-writing-team-handbook.md

justinegeffen committed:
Update operations/research-and-development/product/product-technical-writing-team-handbook.md

justinegeffen committed:
Update operations/research-and-development/product/product-technical-writing-team-handbook.md

wiersgallak commented on operations/research-and-development/product/product-technical-writing-team-handbook.md:
```diff
@@ -221,23 +209,6 @@ Editorial feedback is based on the guidelines laid out in the [Documentation Sty
 
 The [Documentation Style Guide](https://docs.mattermost.com/guides/core.html#documentation-style-guide) and [UI text guidelines](https://docs.mattermost.com/process/documentation-UItext-guidelines.html).
 
-## Customer Education Processes
```
@justinegeffen Wondering if we do want to delete this section.  It still seems fairly relevant and I don't want to lose it.

justinegeffen replied:
I'm happy to keep it. I wasn't sure whether customer education would remain as-is or reworked into something like "Customer Support Processes". 0/5 on renaming. 

Wondering if we could expand on it and incorporate some of the ideas around where and how we manage different types of knowledge as per our discussion.

justinegeffen replied:
I've added the section back into the doc, @wiersgallak. :)

wiersgallak replied:
Thanks - It could be Customer Support Processes, but I am 0/5 as well.  I like it as it is.  I'll approve this PR with changes as it is, but agree we could add the other channels of how we get information on new areas we need to build content for.

justinegeffen replied:
I'm 5/5 on that idea - we can include the #roughdraft and docs process, and forum posts etc. I'll open a new PR for that.

justinegeffen committed:
Update product-technical-writing-team-handbook.md

Re-added customer education content.

justinegeffen committed:
Update product-technical-writing-team-handbook.md

Formatting fixes for bullet list.

justinegeffen commented:
Following up on this, @wiersgallak, I've re-added the CE content and tidied it up slightly.

justinegeffen committed:
Update product-technical-writing-team-handbook.md

Minor updates and added a WIP header to doc release/doc MVP process. Updated Justine's AORs slightly.

wiersgallak reviewed (approved):
I responded to the outstanding question, not blocking to me to merge these changes.
